llvm/utils/TableGen/SDNodeInfoEmitter.cpp

```diff
@@ -324,9 +321,8 @@ static void emitDesc(raw_ostream &OS, StringRef EnumName,
 void SDNodeInfoEmitter::emitDescs(raw_ostream &OS) const {
   StringRef TargetName = Target.getName();
 
-  OS << "#ifdef GET_SDNODE_DESC\n";
-  OS << "#undef GET_SDNODE_DESC\n\n";
-  OS << "namespace llvm {\n";
+  IfDefEmitter IfDef(OS, "GET_SDNODE_DESC");
+  NamespaceEmitter LlvmNs(OS, "llvm");
```

Comment:
```suggestion
  NamespaceEmitter NS(OS, "llvm");
```
like in the other place?